src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -990,6 +1079,20 @@ private boolean validCategory(List<BrAPIScaleValidValuesCategories> categories,
      */
     private String yearsToSeasonDbId(List<String> years, UUID programId) {
```

Comment:
I know this is an existing method, but seems the name and parameters may be misleading, in that you pass in a list of years, but only get back a single seasonDbId.  Maybe this method should just be removed?

Reply:
Done.  (see comment 661aea0b412f535e13f5112b674a719365f70b50)